### ISSUE TITLE
abnormal prostate gland lobe weight 

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3509]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  decreased prostate gland anterior lobe weight/ventral lobe weight/dorsolateral lobe weight
2 . increased prostate gland anterior lobe weight/ventral lobe weight/dorsolateral lobe weight

>2. Synonyms (e.g., absent spleen)

1.  reduced prostate gland anterior lobe weight/ventral lobe weight/dorsolateral lobe weight
2.  increased prostate anterior lobe weight/ventral lobe weight/dorsolateral lobe weight

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  reduction in the average weight of the prostate gland anterior lobe/ventral lobe/dorsolateral lobe in males that secrete part of the seminiferous fluid
2.  increase in the average weight of the prostate gland anterior lobe/ventral lobe/dorsolateral lobe in males that secrete part of the seminiferous fluid

PMID:33368416, PMID:16055444

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  decreased prostate gland weight (MP:0004962)
2.  increased prostate gland weight (MP:0004961)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
parent should be abnormal weight for all terms as increased/decreased refers to average for the whole gland

**Comment by anna-anagnostop** [comment]
Added new terms requested above as well as relevant hypo/hyperplasia terms:

decreased prostate gland anterior lobe weight MP:0031359
prostate gland anterior lobe hypoplasia MP:0031360
increased prostate gland anterior lobe weight MP:0031361
decreased prostate gland ventral lobe weight MP:0031362
prostate gland ventral lobe hypoplasia MP:0031363
increased prostate gland ventral lobe weight MP:0031364
prostate gland ventral lobe hyperplasia MP:0031365
decreased prostate gland dorsolateral lobe weight MP:0031366
prostate gland dorsolateral lobe hypoplasia MP:0031367
increased prostate gland dorsolateral lobe weight MP:0031368
prostate gland dorsolateral lobe hyperplasia MP:0031369
